packages/paste-theme/src/themeProvider.tsx

```diff
@@ -50,18 +56,26 @@ function getProviderThemeProps(theme: ThemeVariants | DeprecatedThemeVariants, c
   }
 }
 
+interface CacheProviderProps {
```

Comment:
Any chance we can grab these option types from `@emotion/cache`?

Reply:
yeah I'll re-export from our styling library from emotion/cache. Thanks gang!